packages/admin-ui/src/pages/support/components/PortsStatusTable.tsx

```diff
@@ -0,0 +1,223 @@
+import ErrorView from "components/ErrorView";
+import Loading from "components/Loading";
+import Ok from "components/Ok";
+import SubTitle from "components/SubTitle";
+import React, { useState } from "react";
+import Button from "components/Button";
+import Table from "react-bootstrap/Table";
+import { ReqStatus } from "types";
+import { useApi } from "api";
+import { api } from "api";
+import { shortNameCapitalized } from "utils/format";
+import {
+  ApiTablePortStatus,
+  PortToOpen,
+  UpnpTablePortStatus
+} from "common/types";
+
+export function PortsStatusTable({
+  isUpnpEnabled
+}: {
+  isUpnpEnabled: boolean;
+}) {
+  const [upnpScanResult, setUpnpScanResult] = useState<UpnpTablePortStatus[]>();
+  const [apiScanResult, setApiScanResult] = useState<ApiTablePortStatus[]>();
+  const [upnpReqStatus, setUpnpReqStatus] = useState<
+    ReqStatus<UpnpTablePortStatus[]>
+  >({});
+  const [apiReqStatus, setApiReqStatus] = useState<
+    ReqStatus<ApiTablePortStatus[]>
+  >({});
+
+  const portsToOpen = useApi.portsToOpenGet();
+
+  async function apiButtonOnClick() {
+    if (portsToOpen.data)
+      try {
+        setApiReqStatus({ loading: true });
+        const apiPorts = await api.portsApiStatusGet({
+          portsToOpen: portsToOpen.data
+        });
+        setApiScanResult(apiPorts);
+        setApiReqStatus({ result: apiPorts });
```

Comment:
The result of the api call is stored twice. Remove useState for `setApiScanResult` and `setUpnpScanResult`